core/vm/exit_code/exit_code.hpp

```diff
@@ -8,6 +8,16 @@
 
 #include <string>
 
+#include "common/outcome.hpp"
+
+namespace fc::vm {
+  enum class VMExitCode : uint8_t {};
```

Comment:
empty enum? is it what you meant?

Reply:
Outcome errors use error enums.
This enum is empty because exit codes have different meaning across vm and different actors.